olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -51,9 +51,10 @@ else()
     find_package(CURL REQUIRED)
 endif()
 
-
 set(Network_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
+option(OLP_SDK_USE_LIBCRYPTO "Enables the libcrypto dependency" ON)
```

Comment:
Should it not be in curl.cmake instead. It does not seem relevant to all other cmake includes.